program/CORE-V IDE prelminary project proposal.md

```diff
@@ -0,0 +1,251 @@
+ - Stable (currently, this repository is a mirror of FSF).
+ - Development (first commits to this repository will have support for CORE-V hardware loop)
+
+
+
+
+## List of project outputs
+
+*Need a list of outputs, i.e.*
+- *Documents to be produced*
+- *Downloadable code to be produced*
+
+
+## TGs Impacted/Resource requirements
+
+This work will be planned and discussed in the SW TG. The project plan will be produced by the project manager with assistance of SW TG, and reviewed/approved in TWG
+
+
+## OpenHW engineering staff resource plan: requirement and availability
+
+- Florian Zaruba - technical contribution and project planning
+- Duncan Bees - program management oversight
+
+## Engineering resource supplied by members - requirement and availability
+
+*The intent here is to write down specific time/person requirements. This is not yet available, and will be supplied in the project plan. As of Preliminary Project Proposal, the major gap identified is a resource for an initial application to test the IDE.*
+
+## OpenHW marketing resource - requirement and availability
+
+- Initial artwork suppplied by Publitek
+- No additional resource requirements yet identified
+
+## Marketing resource supplied by members - requirement and availability
+- No  resource requirements yet identified
+
+## Funding supplied by OpenHW - requirement and availability 
+- No funding requirements yet identified
+
+
+## Funding supplied by members - requirement and availability
+- No funding requirements yet identified
+
+
+
+## Architecture diagram
+*Need diagram showing SW architecture and also real-world physical setup for intended use. Action to Alexander* 
+
+## Who would make use of OpenHW output
+CORE-V CDT will be used by developers created code for the following target platforms
+- CV32E40P FPGA platform
+- *expand/detail this list*
+
+## Project license model
+
+This project will be released under EPL 2.0
+* Explain how the code under other licenses, such as GNU PL, will be incorporated *
+
+## Description of initial code contribution, if required
+* Explain initial source for all code contributions, and which initial code contributions will have Contribution Questionnaire generated *
+
+
+## Repository Structure
+* Explain the initial OpenHW repository structure and how it will be migrated to Eclipse Foundation management*.
+
+
+
+
+## Project distribution model
+
+The project code will be distributed using an Eclipse p2 repository and an archive for each target platform.
+
+* To be discussed with Mikael Barbero and described here*
+
+
+## Preliminary Project plan
+
+The following are initial steps forming elements of a project plan. A full project plan is not yet available.
+
+*Note, at Preliminary Project Launch we don't need a full project plan but we should enumerate key steps and timelines where available*
+
+### Steps to First Integration at end of 2020 (not neccesarily in the following order)

```

Comment:
I observe that some of these steps are *much* bigger than others. It might be useful to draw out which are big steps.

Reply:
restructured and simplified